smoke-test.sh

```diff
@@ -182,6 +179,9 @@ cd "$testingDir"
 
 # Unzip dotnet if the dotnetDir is not specified
 if [ "$dotnetDir" == "" ]; then
+    OUTPUT_DIR="$SCRIPT_ROOT/bin/x64/Release/"
+    DOTNET_TARBALL=$(ls ${OUTPUT_DIR}dotnet-sdk-${VERSION_PREFIX}*)
```

Comment:
While you're moving this, might as well add quotes around the `ls` arg.

Reply:
Fixed.